AGENTS.md

```
@@ -149,12 +149,12 @@ Development infrastructure template repository providing DevContainer images, CI
-| Script | Command | Purpose |
-| ----------------------------------------------------------------------------------------------------------------------- | ---------------------------------- | -------------------------- | ----------------------- |
-| `format:check` | `prettier --check .` | Code formatting validation |
-| `lint` | `eslint . --ext .js` | Code quality validation |
-| `test` | `jest --runInBand` | Unit test execution |
-| `shellcheck` | `find script -name '\*.sh' -type f | xargs -r shellcheck -x` | Shell script validation |
+| Script                                                                                                                  | Command                           | Purpose                    |
+| ----------------------------------------------------------------------------------------------------------------------- | --------------------------------- | -------------------------- |
+| `format:check`                                                                                                          | `prettier --check .`              | Code formatting validation |
+| `lint`                                                                                                                  | `eslint . --ext .js`              | Code quality validation    |
+| `test`                                                                                                                  | `jest --runInBand`                | Unit test execution        |
+| `shellcheck`                                                                                                            | `find script -name '*.sh' -type f | xargs -r shellcheck -x`    | Shell script validation |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)